packages/fiori/src/NotificationListGroupItem.hbs

```diff
@@ -3,7 +3,7 @@
 	@focusin="{{_onfocusin}}"
 	@focusout="{{_onfocusout}}"
 	@keydown="{{_onkeydown}}"
-	role="option"
+	role="listitem"
```

Comment:
Also use the accessibleRole attribute for the ui5-list on line 80 in this file.

Reply:
yes, thanks for spotting it